ubuntudesign/vanilla-framework
# Update code block styles

## Done
Removed the dark code snippet styling and updated the code blocks to match the latest design.

## QA
- Run `gulp jekyll`
- Check the following patterns match the design
 - http://127.0.0.1:4000/vanilla-framework/examples/base/code/
 - http://127.0.0.1:4000/vanilla-framework/examples/patterns/code-numbered/
 - http://127.0.0.1:4000/vanilla-framework/examples/patterns/code-snippet/

## Details
Fixes https://github.com/ubuntudesign/vanilla-framework/issues/818

**Note** - Removal of `p-code-snippet--dark`, replace with `p-code-snippet`
